lanweatherd/src/main.cpp

```diff
@@ -84,16 +84,16 @@ int main(void) {
 
     vector<thread> threads;
 
-    thread thr_nws_fetch(nws_loop, ref(cache));
+    std::thread thr_nws_fetch(nws_loop, ref(cache));
```

Comment:
Because we're `using namespace std` we shouldn't need to explicitly say `std::thread` and just `thread` should work, but I'm not opposed to dropping the namespace too. That would require making line 85 `std::vector` too.

Reply:
Ah you're right! `std::` is not necessary here since we declare `using namespace std` above. Admittedly, I made these changes a while back, so I figured I added `std::` to fix a broken build. Thanks for pointing that out.